server/startrabbit.sh

```diff
@@ -17,9 +17,16 @@ else
 			rabbitmqctl join_cluster --ram rabbit@$CLUSTER_WITH
 		fi
 		rabbitmqctl start_app
+                
+        if [ "$RABBITMQ_DEFAULT_USER" -a "$RABBITMQ_DEFAULT_PASS"] then
+			rabbitmqctl wait /var/run/rabbitmq/pid
```

Comment:
Looks like this won't work - since we are starting in a detached mode a pid file is not created, hence this statement will end up hanging forever waiting for the file to appear.